plugins/woocommerce/tests/e2e-pw/tests/merchant/page-loads.spec.js

```diff
@@ -87,8 +87,8 @@ const wcPages = [
 			{
 				name: 'Overview',
 				heading: 'Overview',
-				element: '.woocommerce-marketing-card-header-description',
-				text: 'Start by adding a channel to your store',
+				element: '.woocommerce-marketing-channels-card',
+				text: 'Channels',
```

Comment:
based on the discussion here https://github.com/woocommerce/woocommerce/pull/51744#issuecomment-2385540520